core/modules/macros/uniquetitle.js

```diff
@@ -0,0 +1,34 @@
+/*\
+title: $:/core/modules/macros/uniquetitle.js
+type: application/javascript
+module-type: macro
+Macro to return a unique title given a base name
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+/*
+Information about this macro
+*/
+
+exports.name = "uniquetitle";
+
+exports.params = [
+	{name: "baseName"},
+	{name: "options"}
+];
+
+/*
+Run the macro
+*/
+exports.run = function(baseName, options) {
+	if (!baseName) {
```

Comment:
Our coding style is not to have any white space between `if` and the following opening bracket.

Reply:
I have been working on too many different projects lately. It is fixed.

Reply:
I sympathise! I'd love to have automated tooling to check/fix the simple stuff.